src/handlers/comment/handlers/index.ts

```diff
@@ -215,20 +214,87 @@ export const issueReopenedBlameCallback = async (): Promise<void> => {
     logger.info(`Found ${pullDiff.files.length} files changed in commit ${matchingPull?.merge_commit_sha}`);
   }
 
-  const comment = `
-  The following pull requests were merged after this issue was closed:
-  ${onlyPRsNeeded
-    .sort()
-    .map((pullNumber) => `<ul><li>#${pullNumber}</li></ul>`)
-    .join("\n")}
+  const matchingSlice = matchingPull?.merge_commit_sha?.slice(0, 8);
+  const currentSlice = currentCommitSha.slice(0, 8);
 
-  | **Files Changed** | **Commit** | **Author** |
-  | --- | --- | --- |
-  ${onlyPRsNeeded.map((_pullNumber, index) => `| ${fileLens[index]} | ${commitsThatMatch[index].sha} | ${commitsThatMatch[index].author?.login} |`).join("\n")}
+  const twoDotUrl = `<code>[${matchingSlice}..${currentSlice}](${repository.html_url}/compare/${matchingPull?.merge_commit_sha}..${currentCommitSha})</code>`;
+
+  interface Blamer {
+    author: string;
+    count: number;
+  }
 
-  Basehead Comparison: ${matchingPull?.merge_commit_sha}...${currentCommitSha}
+  const blamers: Blamer[] = [];
+
+  for (const diff of diffs) {
+    if (!diff.files) continue;
+    for (const file of diff.files) {
+      const linesChanged = file.patch?.split("\n").filter((line) => line.startsWith("+")).length;
+      const author = diff.base_commit?.author?.login;
+      const blamer = blamers.find((b) => b.author === author);
+      if (blamer) {
+        blamer.count += linesChanged || 0;
+      } else {
+        blamers.push({ author: author || "", count: linesChanged || 0 });
+      }
+    }
+  }
+
+  const advancedBlameTable = `
+  | **Blame** | **Count** | **%** |
+  | --- | --- | --- |
+  ${Array.from(new Set(blamers))
+    .filter((blamer) => blamer.count > 0)
+    .sort((a, b) => b.count - a.count)
+    .map((blamer) => {
+      const linesChanged = blamer.count;
+      const totalLinesChanged = blamers.reduce((acc, val) => acc + val.count, 0);
+      const percent = Math.round((linesChanged / totalLinesChanged) * 100);
+      return `| ${blamer.author} | ${blamer.count} | ${percent}% |`;
```

Comment:
We don't need to display the count but the percentage is a good idea. Its a nice abstraction. The only thing that people will pay attention to is comparing to each other who is more likely at fault lol